Formula/verilator.rb

```diff
@@ -24,6 +24,8 @@ class Verilator < Formula
   end
 
   depends_on "python@3.9" => :build
+  uses_from_macos "bison"
```

Comment:
Can you add a space between the depends_on and uses_from_macos. We have no audit for this but usually we separate them a little bit.